BE/src/test/java/com/ahoo/airbnb/reservation/ReservationServiceTest.java

```diff
@@ -0,0 +1,86 @@
+package com.ahoo.airbnb.reservation;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.BDDMockito.given;
+
+import com.ahoo.airbnb.data.TestData;
+import com.ahoo.airbnb.exception.ExceptionMessage;
+import com.ahoo.airbnb.reservation.dtos.ChargesResponse;
+import com.ahoo.airbnb.reservation.dtos.RoomChargeRequest;
+import com.ahoo.airbnb.reservation.dtos.RoomChargeResponse;
+import com.ahoo.airbnb.room.RoomRepository;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.transaction.annotation.Transactional;
+
+@ExtendWith(MockitoExtension.class)
+@Transactional
+class ReservationServiceTest {
+
+    @Mock
+    RoomRepository roomRepository;
+
+    @InjectMocks
+    ReservationService reservationService;
+
+    @Test
+    @DisplayName("존재하는 roomId에 해당하는 숙소의 1박당 요금, 총 요금, 상세 요금 항목들을 계산하여 리턴한다.")
+    void calculateTotalChargeOfSuccessTest() {
```

Comment:
한글로 작성해주셔도 됩니다. 독자 대상이 한국어 사용자일 것이므로, 지금은 영어 대신 한국어를 사용합시다.
```suggestion
    void roomId에_해당하는_요금을_계산가능하다() {
```

Reply:
읽기에도 좋고, 못하는 영작 때문에 고민도 안해도 되고 너무 좋네요ㅎㅎ 한글메서드명으로 변경하였습니다 감사합니다 😆